source/language/gates.rst

```diff
@@ -319,9 +321,11 @@ and :math:`N_U` is the negative controlled-:math:`U` gate.
        negctrl @ x q1, q2;
    }
 
-``ctrl`` and ``negctrl`` both accept an optional positive integer argument ``n``, specifying the
-number of control bits (omission means ``n=1``). ``n`` must be a compile-time constant. For an ``N``
-qubit operation, these operations are mathematically defined as
+``ctrl`` and ``negctrl`` both accept an optional positive integer parameter ``n``, specifying the
+number of control arguments (omission means ``n=1``). The sum of all control modifier parameters
+must be equal to the count of all control arguments, otherwise the statement is an invalid.
```

Comment:
I think the inserted sentence here might be a bit more complicated than just not having it - with the bad examples corrected, I think the discussion above of "`ctrl(n)` adds `n` arguments to the list" is sufficient to explain this.  Introducing new terminology like "sum of control modifier parameters" etc makes it seem to me like it's more complicated than it really is (like it makes me double-take over whether `ctrl` behaves differently to `ctrl(1)` because the former doesn't have a "modifier parameter").

Reply:
Ok, I deleted this phrase